### PR TITLE
add leaflet fullscreen functions to map class

### PR DESCRIPTION
**Pull request by bjubes**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/Leaflet/Leaflet.fullscreen#usage
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header. (adds features from 1.0.0 so didn't bump version number to 1.0.2)

### CODE REVIEW

File: types/leaflet-fullscreen/index.d.ts [hunk 1]
```diff
@@ -10,4 +11,8 @@ declare module 'leaflet' {
     interface MapOptions {
         fullscreenControl?: true | {pseudoFullscreen: boolean} | undefined;
     }
+    interface Map {
+        isFullscreen: () => boolean;
+        toggleFullscreen: () => void;
```
Comment (peterblazejewicz) [inline types/leaflet-fullscreen/index.d.ts:16]:
```ts
        toggleFullscreen: () => void;
```
Should this accept optional options?:
https://github.com/Leaflet/Leaflet.fullscreen/blob/gh-pages/dist/Leaflet.fullscreen.js#L59

Comment (bjubes) [reply types/leaflet-fullscreen/index.d.ts]: Yes. Just added the a type for the options and made it an optional arg to to `toggleFullscreen()`

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@bjubes thx!

**Comment by bjubes** [comment]
Ready to merge

**Pull request merged**